pkg/api/validation/validation.go

```diff
@@ -4044,7 +4045,7 @@ func validateEndpointPort(port *api.EndpointPort, requireName bool, fldPath *fie
 	} else if len(port.Name) != 0 {
 		allErrs = append(allErrs, ValidateDNS1123Label(port.Name, fldPath.Child("name"))...)
 	}
-	for _, msg := range validation.IsValidPortNum(int(port.Port)) {
+	for _, msg := range validation.IsValidPortNumWithZeroInclusive(int(port.Port)) {
```

Comment:
wait, why do we need to allow zero?  If they specified an EPP block, they should be forced to specify a valid port, right?